screens/PrepareFace.tsx

```diff
@@ -16,34 +141,56 @@ const PrepareFace: FunctionComponent = () => {
   if (!permissions) {
     return <Text>Please give permissions</Text>;
   }
+  const inBoundingBox = ({ bounds: { size, origin } }: Face): boolean => {
+    return (
+      origin.x + size.width / 4 >= boundingBox.left &&
+      origin.x + (size.width * 3) / 4 <= boundingBox.left + boundingBox.width &&
+      origin.y + size.height / 4 >= boundingBox.top &&
+      origin.y + (size.height * 3) / 4 <= boundingBox.top + boundingBox.height
+    );
+  };
   return (
     <View style={Styles.container}>
       <Camera
         style={Styles.camera}
         type={Camera.Constants.Type.front}
-        onFacesDetected={(face) => setSeeingFace(face.faces.length > 0)}
+        onFacesDetected={(face) =>
+          setSeeingFace(face.faces.some(inBoundingBox))
+        }
         flashMode={Camera.Constants.FlashMode.auto}
       />
-      <View style={Styles.bottomRow}>
-        <Ionicons.Button
-          style={Styles.button}
-          name={seeingFace ? "arrow-forward-circle" : "hand-left"}
-          color={!seeingFace ? "red" : "black"}
-          onPress={() => {
-            //@ts-ignore
-            navigation.replace("Camera");
-          }}
-          disabled={!seeingFace}
-        >
-          {seeingFace
-            ? "Begin Recording"
-            : "Please Show Your Face to Begin Recording"}
-        </Ionicons.Button>
-      </View>
+      <View
+        style={InViewRectStyle(
+          boundingBox.left,
+          boundingBox.top,
+          boundingBox.width,
+          boundingBox.height,
+          seeingFace
+        )}
+      />
+      <BottomInfoPrompt inScreen={seeingFace} finished={finished} />
     </View>
   );
 };
 
+const InViewRectStyle = (
+  left: number,
+  top: number,
+  width: number,
+  height: number,
+  seen: boolean
+): StyleProp<ViewStyle> => ({
+  position: "absolute",
+  left: left,
+  top: top,
+  width: width,
+  height: height,
+  borderWidth: 3,
+  borderStyle: "dashed",
+  borderColor: seen ? "#00A51A" : "#A50000",
```

Comment:
thinking we should pull out the colors to be constants at some point or another

Reply:
Definitely agree! Constants are _wavy_, and I will do this.